homeassistant/const.py

```diff
@@ -248,6 +247,10 @@
 DEVICE_CLASS_TIMESTAMP: Final = "timestamp"
 DEVICE_CLASS_VOLTAGE: Final = "voltage"
 
+# ### STATE CLASSES ####
+STATE_CLASS_MEASUREMENT: Final = "measurement"
```

Comment:
This is already available here:
https://github.com/home-assistant/core/blob/1cc3ffe20dc4b03d816bf9f68a37bc42e0df2f5d/homeassistant/components/sensor/__init__.py#L70

Reply:
oh, well. I will try to change it to these then.

Reply:
done